subway_structure.py

```diff
@@ -1741,8 +1748,7 @@ def validate_overground_lines(self):
             ]
         )
         if self.found_tram_lines != self.num_tram_lines:
-            self.error_if(
-                self.found_tram_lines == 0,
+            self.error(
```

Comment:
This might raise errors when there are tram lines not in list.

Reply:
Returned "error-if" logic in this case.